salt/modules/dpkg.py

```diff
@@ -274,7 +306,7 @@ def _get_pkg_info(*packages, **kwargs):
     ret = []
     cmd = "dpkg-query -W -f='package:" + bin_var + "\\n" \
           "revision:${binary:Revision}\\n" \
-          "architecture:${Architecture}\\n" \
+          "arch:${Architecture}\\n" \
```

Comment:
IIRC this should not be changed to `arch`, otherwise we won't get the right architecture value here: https://github.com/saltstack/salt/blob/576e8717501e1cec0e2dd968526eb14ec4dc5f0e/salt/modules/dpkg.py#L342 

Am I missing something?

Reply:
@meaksh sure you missing something:

```
"meaksh:${Architecture}\\n"
```
This is also valid. :wink: That said:
```
$ salt-call --local pkg.info_installed vim attr=arch,build_date,install_date,version
local:
    ----------
    vim:
        ----------
        arch:
            amd64
        build_date:
            2016-04-03T13:25:38Z
        install_date:
            2017-11-02T09:55:37Z
        version:
            2:7.4.1689-3ubuntu1.2
```
Looks OK?